src/common/utils.ts

```diff
@@ -168,3 +168,16 @@ export const toggleDebugMode = (toggleBtn: HTMLElement): void => {
   Debugger.DebugMode = isOn;
   toggleBtn.innerText = "Dev";
 };
+
+export const hideShowElement = (
+  isHide: boolean = false,
+  element: HTMLElement
+) => {
+  if (isHide) {
+    element.classList.remove("show");
+    element.style.zIndex = "0";
+  } else {
+    element.classList.add("show");
+    element.style.zIndex = "999";
+  }
```

Comment:
Why do we need to change the zIndex here if we are removing the class? Can the Z index be included in that show class?